torch_frame/typing.py

```diff
@@ -44,6 +44,9 @@ class NAStrategy(Enum):
     MEAN = 'mean'
     MOST_FREQUENT = 'most_frequent'
     ZEROS = 'zeros'
+    MOST_RECENT_TIMESTAMP = 'most_recent_timestamp'
+    LEAST_RECENT_TIMESTAMP = 'least_recent_timestamp'
+    MEDIAN_TIMESTAMP = 'median_timestamp'
```

Comment:
I think we can have MEDIAN (which is more general)?

Reply:
okay. Let me create a different pr to refactor.